### PR TITLE
Add firebase

### PR DESCRIPTION
**Pull request by thepushkarp**
Add firebase to project

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,5 +12,28 @@
...
+
+  <!-- Add Firebase products that you want to use -->
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-auth.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-firestore.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-database.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-storage.js"></script>
+
+  <script>
+    // Your web app's Firebase configuration
+    var firebaseConfig = {
```
Comment (aashutoshrathi) [inline src/index.html]: Can we get this config somewhere safe?

Comment (parth181195) [reply src/index.html]: @thepushkarp we should angularfire for firebase instade of firebase.js

and for that config make one ts file and put it there and add it to ignore

Comment (thepushkarp) [reply src/index.html]: Sure @aashutoshrathi @parth181195

**Comment by thepushkarp** [comment]
Sir, is this config okay?
I've added the config in /src/app/environments/environmentConfig.ts and added it to gitignore.

**Comment by thepushkarp** [comment]
image
image


Basic Google Login works and and the data is stored to a `users` collection in firebase.

**Comment by aashutoshrathi** [comment]
This LGTM. Proceed quick

**Comment by aashutoshrathi** [comment]
@parth181195 Please merge this.

**Comment by thepushkarp** [comment]
Done

**Comment by aashutoshrathi** [comment]
@samagragupta No need to squash, just test it once. I'll do squash and merge.

**Comment by aashutoshrathi** [comment]
@samagragupta Is it working, properly.
Also send me the config.

**Comment by thepushkarp** [comment]
@aashutoshrathi I've mailed you the config.

**Comment by aashutoshrathi** [comment]
Add `@angular/fire": "^5.2.1",` to your package.json
`npm i `and then push again

**Pull request merged**